btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/provider/BroadcastsProvider.kt

```diff
@@ -21,6 +23,7 @@ class BroadcastsProvider(
     private val withdrawalQueryHelper: IrohaQueryHelper
 ) {
 
+    private val gson = Gson()
```

Comment:
how about using a separate `static` `Gson` somewhere from utility class in all the classes need this

Reply:
Good idea